freqtrade/rpc/telegram.py

```diff
@@ -74,10 +75,48 @@ def __init__(self, freqtrade) -> None:
 
         self._updater: Updater
         self._config = freqtrade.config
+        self._validate_keyboard()
         self._init()
         if self._config.get('fiat_display_currency', None):
             self._fiat_converter = CryptoToFiatConverter()
 
+    def _validate_keyboard(self) -> None:
+        """
+        Validates the keyboard configuration from telegram config
+        section.
+        """
+        self._keyboard: List[List[Union[str, KeyboardButton]]] = [
+            ['/daily', '/profit', '/balance'],
+            ['/status', '/status table', '/performance'],
+            ['/count', '/start', '/stop', '/help']
+        ]
+        # do not allow commands with mandatory arguments and critical cmds
+        # like /forcesell and /forcebuy
+        # TODO: DRY! - its not good to list all valid cmds here. But otherwise
+        #       this needs refacoring of the whole telegram module (same
+        #       problem in _help()).
+        valid_keys: List[str] = ['/start', '/stop', '/status', '/status table',
+                                 '/trades', '/profit', '/performance', '/daily',
+                                 '/stats', '/count', '/locks', '/balance',
+                                 '/stopbuy', '/reload_config', '/show_config',
+                                 '/logs', '/whitelist', '/blacklist', '/edge',
+                                 '/help', '/version']
+
+        # custom shortcuts specified in config.json
+        cust_keyboard = self._config['telegram'].get('keyboard', [])
+        if cust_keyboard:
+            # check for valid shortcuts
+            invalid_keys = [b for b in chain.from_iterable(cust_keyboard)
+                            if b not in valid_keys]
+            if len(invalid_keys):
+                logger.warning('rpc.telegram: invalid commands for custom '
+                               f'keyboard: {invalid_keys}')
+                logger.info('rpc.telegram: using default keyboard.')
```

Comment:
Considering that configuring your keyboard this should be a "one-off" activity (you do that once - until you like how it looks), i think we should maybe have the bot fail in this case (`raise OperationalException('...')`).

As for the log - you shouldn't use `rpc.telegram:` - this is automatic by the logging module ... now it looks as follows, having rpc.telegram twice:

```
2020-12-22 07:29:12,227 - freqtrade.rpc.telegram - WARNING - rpc.telegram: invalid commands for custom keyboard: ['/reload_co1nfig']
2020-12-22 07:29:12,227 - freqtrade.rpc.telegram - INFO - rpc.telegram: using default keyboard.
```

Reply:
throwing an OperatinalException looks confusing in the logs.
`2020-12-22 12:20:03,907 - freqtrade.commands.trade_commands - ERROR - invalid commands for custom keyboard: ['/dailyNOW']`

`freqtrade.commands.trade_commands` makes no sense in my opinion.

Any suggestions ?